engine/src/main/java/org/terasology/monitoring/gui/ChunkMonitorDisplay.java

```diff
@@ -62,18 +62,13 @@
 @SuppressWarnings("serial")
 public class ChunkMonitorDisplay extends JPanel {
 
-    public static final Color COLOR_COMPLETE = new Color(0, 38, 28);
-    public static final Color COLOR_INTERNAL_LIGHT_GENERATION_PENDING = new Color(4, 76, 41);
-    public static final Color COLOR_ADJACENCY_GENERATION_PENDING = new Color(150, 237, 137);
+    private static final Color COLOR_COMPLETE = new Color(0, 38, 28);
+    private static final Color COLOR_INTERNAL_LIGHT_GENERATION_PENDING = new Color(4, 76, 41);
+    private static final Color COLOR_HIGHLIGHT_TESSELLATION = Color.blue.brighter().brighter();
+    private static final Color COLOR_SELECTED_CHUNK = new Color(255, 102, 0);
+    private static final Color COLOR_DEAD = Color.LIGHT_GRAY;
 
-    public static final Color COLOR_HIGHLIGHT_TESSELLATION = Color.blue.brighter().brighter();
-
-    public static final Color COLOR_SELECTED_CHUNK = new Color(255, 102, 0);
-
-    public static final Color COLOR_DEAD = Color.lightGray;
-    public static final Color COLOR_INVALID = Color.red;
-
-    private static final Logger logger = LoggerFactory.getLogger(ChunkMonitorDisplay.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ChunkMonitorDisplay.class);
```

Comment:
All caps logger or not is a tricky subject. See https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case - generally we've kept it lower case so probably best to stay consistent that way, at least for now :-)

Reply:
Indeed, according to _Java Code Style_ is should be lower-case. I changed it after CheckStyle scan, however CheckStyle cannot check if this object reference is never followed by a ".".